programs/domain-registry/src/state.rs

```diff
@@ -2,6 +2,7 @@ use anchor_lang::{prelude::*, solana_program::system_instruction};
 use bytemuck::{Pod, Zeroable};
 use fogo_sessions_sdk::session::AuthorizedProgram;
 use std::marker::PhantomData;
+use std::mem::size_of;
```

Comment:
Added this because I was having some trouble getting nightly to work on nix and I don't want to deal with it right now, this unblocks me